IGC/AdaptorOCL/PreprocessSPVIR.cpp

```diff
@@ -79,7 +79,7 @@ Value* PreprocessSPVIR::getWidenImageCoordsArg(Value* Coords)
     Value* newCoords = nullptr;
     if (!isa<VectorType>(coordsType))
     {
-        Value* undef = UndefValue::get(VectorType::get(coordsType, 4));
+        Value* undef = UndefValue::get(FixedVectorType::get(coordsType, 4));
```

Comment:
Please use `FixedVectorType` wrapper from `IGCLLVM` namespace defined here: https://github.com/intel/intel-graphics-compiler/blob/master/IGC/WrapperLLVM/include/llvmWrapper/IR/DerivedTypes.h#L23

This allows for LLVM <= 10 backwards compatibility.

Reply:
@pszymich, sure. I've changed the calls now.